SimplenoteIntents/ResolutionResults/IntentNote+Helpers.swift

```diff
@@ -21,6 +21,31 @@ extension IntentNoteResolutionResult {
 
         return IntentNoteResolutionResult.success(with: intentNote)
     }
+
+    static func resolveIntentNote(for content: String, in coreDataWrapper: ExtensionCoreDataWrapper) -> IntentNoteResolutionResult {
+        guard let notes = coreDataWrapper.resultsController()?.notes() else {
+            // TODO: Better error
+            return IntentNoteResolutionResult.unsupported()
+        }
+        let filteredNotes = notes.filter({ $0.content?.contains(content) == true })
```

Comment:
Ideaaaally this happens at the Core Data level.

I haven't seen massively big accounts, but this will load all the things in memory, and run the lookup there.

Reply:
@charliescheer ^ Good to disregard though, I don't think this should be a problem in Simplenote

Reply:
I'm gonna disregard it for now I think.  If we get any complaints then we can optimize